server/fixtures/auth.js

```diff
@@ -0,0 +1,26 @@
+import jwt from 'jsonwebtoken';
+import { User } from '../db/models';
+
+const authenticate = (req, res) => {
+  User.findOne({
+    where: {
+      email: req.body.email,
+    }
+  }).then((user) => {
+    const payLoad = {
+      id: user.id,
+      email: user.email,
+    };
+    const token = jwt.sign(payLoad, 'secret', { expiresIn: '1hr' });
+    res.status(200).json({
+      message: 'Success',
+      token,
+    });
+
+    // res.status(200).json({ status: 'Success', token, });
```

Comment:
#### Nitpick
Remove commented code lines/blocks. If it's not needed, delete it.